velox/vector/tests/VectorTest.cpp

```diff
@@ -45,6 +45,11 @@ class TestingLoader : public VectorLoader {
       return;
     }
     *result = data_;
+    rowCounter_ += rows.size();
+  }
+
+  int32_t getRowCount() {
```

Comment:
nit: by convention we do not use 'get' prefixes for getters and make them 'const'

```
int32_t rowCount() const
```